scripts/masterCron.ts

```diff
@@ -113,14 +136,28 @@ async function pushTokenListToGit(tokenlist: string, fileName: string) {
   // when setting all options in a single object
   const git: SimpleGit = simpleGit(options);
 
-  fs.writeFile(`../ribbon-token-list/${fileName}`, tokenlist);
+  let newTokenSet = tokenSet;
+
+  let currentTokenSet = (
+    JSON.parse(fs.readFileSync(`../ribbon-token-list/${fileName}`)) as TokenSet
+  ).tokens;
+
+  // add new week's otokens to token list and remove duplicates
+  newTokenSet.tokens = Array.from(
+    new Set(currentTokenSet.concat(newTokenSet.tokens))
```

Comment:
Just confirming, does `Set` filter out the duplicate objects? Normally only works when it's not an object